toxic-review-warning/src/script.js

```diff
@@ -0,0 +1,131 @@
+const url = new URL('./worker.js', import.meta.url);
+import { MESSAGE_CODE, MODEL_STATUS } from './consts.js';
+import { TYPING_DELAY } from './config.js';
+
+// Initialization
+
+let modelStatus = MODEL_STATUS.NOT_STARTED;
+let isToxicityVisible = false;
+updateToxicityVisibility(isToxicityVisible);
+displayModelStatus(modelStatus);
+updateToxicityAssessmentDisplay({ isToxic: false, toxicityTypeList: [] });
+const worker = new Worker(url);
+let typingTimeout = 0;
+
+// DOM manipulation, display and events
+
+function updateToxicityVisibility(isVisible) {
+  isToxicityVisible = isVisible;
+  document.getElementById('aiWrapper').className = isToxicityVisible
+    ? 'visible'
+    : 'hidden';
+}
```

Comment:
Can this use just one class  like `visible` that you then [`toggle()`](https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList/toggle), or, maybe better, just the `hidden` attribute, since this is what you want in the end, the thing to be shown or not.

Reply:
A simple `toggle` wouldn't do it, since this visibility depends on multiple factors - but TIL `toggle` takes a second argument (`...`, boolean) which is convenient and does simplify the code a bit 👍 Done.

(Note: I like classes a bit better than `hidden` in general, since any `display:` CSS property overrides the `hidden` attribute, and I feel like I end up needing to add a new wrapper and tweak my layout.)